packages/marshal/src/encodeToSmallcaps.js

```diff
@@ -386,6 +386,7 @@ export const makeDecodeFromSmallcaps = (decodeOptions = {}) => {
               encoding,
               decodeFromSmallcaps,
             );
+            // @ts-ignore FIXME tsc thinks result is 'copyRecord' style
```

Comment:
What’s the fix? Is this as good as it gets?

Reply:
clarified